Automattic/wp-calypso
# Plans 2023: Migrate p2 pricing page grid

**Note:**
Change branch base before merging

Related to https://github.com/Automattic/wp-calypso/issues/77652

## Proposed Changes

* Remove PlansFeatures component for p2 pricing page
* Replace with PlansFeaturesMain component

## Testing Instructions



* Check out this branch
* `yarn` && `yarn start`
* Navigate to calypso.localhost:3000/start/p2
* Create a new P2 workspace
* Navigate to calypso.localhost:3000/plans/{P2_SITE_SLUG}
* Verify that the new PlansFeaturesMain component is being used by comparing with production

## Pre-merge Checklist
